### PR TITLE
Julie needs to post Turtle Post

### PR DESCRIPTION
**Pull request by juliees39**
Here is the link to my site: http://juliees39.github.io/spring2014/

I can't get the post to show up. I'm hoping it's just because it needs more time because I can't find any mistakes in the code...

### CODE REVIEW

**Comment by eah13** [comment]
@juliees39 please post a link to your working site with this code.

**Comment by eah13** [comment]
@juliees39 the file doesn't have .md on the end!  It's a little thing that makes a big difference.

**Comment by juliees39** [comment]
@eah13  Okay I fixed it.

File: _posts/2014-01-16-julieturtle.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+   carlos.color(aColor)
+   carlos.forward(50)
+   carlos.left(90)
+   carlos.forward(-50)
+   
+wn.exitonclick()
+
+```
+
+![Turtle Image](http://www.flickr.com/photos/63655989@N03/11988980026/)
```
Comment (eah13) [inline _posts/2014-01-16-julieturtle.md]: This needs to be a link to an image file.  Right click and open the image itself in another window until you get a `.png`, `.jpg` etc on the end of the URL.

**Comment by eah13** [comment]
@juliees39 Did you visit the post on your site?  The image looks broken to me:
http://juliees39.github.io/spring2014/2014/01/16/julieturtle.html

I left a comment for you inline about where I think the fix will be.  Should be one more fix to that URL then you're ready to go!

**Comment by juliees39** [comment]
@eah13  Okay I fixed is so I think it is finally all set.

**Comment by eah13** [comment]
Hey alright.  Nice abstract picture.

Good job getting these resolved.  In the future remember to use your site as a tool to see if your code is rendering correctly!

**Pull request merged**